src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -79,16 +75,13 @@ const VideoPlayer = ({
     if (video) {
       // Instantiate the player and keep the instance in state
       setPlayer(
-        await createPlayer(
+        createPlayer(
```

Comment:
I think there is no need to use `useAsyncEffect` hook here. You can use `useEffect`